sdk/core/Azure.Core.TestFramework/src/TestRecording.cs

```diff
@@ -196,10 +196,12 @@ public TestRandom Random
 #if NET6_0_OR_GREATER
                             var liveSeed = RandomNumberGenerator.GetInt32(int.MaxValue);
 #else
+#pragma warning disable SYSLIB0023
```

Comment:
This is external code, I dont think we should be changing it

Reply:
Unfortunately, without this change, building will fail.